.github/workflows/test_apps.yml

```diff
@@ -0,0 +1,56 @@
+name: TestApps
+
+on:
+  push:
+    branches: [ main ]
+    paths-ignore:
+    - '*.md'
+    - 'LICENSE'
+    - '*.podspec'
+    - 'Package*.swift'
+    - 'Source/SwiftPackage/**'
+    - '.github/workflows/cocoapods.yml'
+    - '.github/workflows/swiftpm.yml'
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+    - '*.md'
+    - 'LICENSE'
+    - '*.podspec'
+    - 'Package*.swift'
+    - 'Source/SwiftPackage/**'
+    - '.github/workflows/cocoapods.yml'
+    - '.github/workflows/swiftpm.yml'
+
+jobs:
+  xcodebuild:
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        # iOS runs into codesigning requirements to build applications, haven't
+        # found a good way to suppress that and just ensure things build.
+        #PLATFORM: ["iOS", "macOS"]
```

Comment:
I tried a few things, but failed to find a way to get Xcode to build the iOS app and skip signing/do adhoc/etc.